allauth/socialaccount/providers/openid_connect/provider.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+from allauth.account.models import EmailAddress
+from allauth.socialaccount import app_settings
+from allauth.socialaccount.providers.base import ProviderAccount
+from allauth.socialaccount.providers.oauth2.provider import OAuth2Provider
+
+
+class OpenIDConnectProviderAccount(ProviderAccount):
+    def to_str(self):
+        dflt = super(OpenIDConnectProviderAccount, self).to_str()
+        return self.account.extra_data.get("name", dflt)
+
+
+class OpenIDConnectProvider(OAuth2Provider):
+    id = "openid-connect"
```

Comment:
The convention is that the `id` matches the package name, `openid_connect` in this case. Though, perhaps it would be best to rename the package (and the `id`) to `oidc`?

Reply:
I changed the provider ID to `openid_connect` as suggested. I'm happy to change it to `oidc` if folks feel strongly. If we were to rename to `oidc`, the `openid_connect` provider directory should also be renamed to `oidc`.